### ISSUE TITLE
Iridium always makes persistent connection to "accounts.google.com"

### ISSUE THREAD

**Issue by ghost** [iridium-browser/tracker#250]
Hi,

When starting Iridium, it will make a DNS request for "accounts.google.com" and create a connection to a Google IP that appears to last for the length of the browsing session. All traffic is sent over TLS, so I am unable to see what it is, but it sends data fairly consistently, so I do not think it is a simple check-in. My setup is the defaults plus safe browsing and ad-blocking turned off.  

Edit: It may just be a check-in of some sort. I tested 3 more times and saw a flurry of traffic on startup and then little after that. 
This is with version 2019.04 on OpenBSD.

**Comment by byteskeptical** [comment]
I can confirm this behavior with safe browsing, ad-blocking disabled as well:
```
02:38:21.457344 localhost.29373 > localhost.domain: 42190+ AAAA? accounts.google.com.(37)
02:38:21.794595 localhost.domain > localhost.29373: 42190 1/0/0 AAAA mia09s20-in-x0d.1e100.net(65)
02:38:21.794852 localhost.34780 > localhost.domain: 56477+ A? accounts.google.com.(37)
02:38:21.860932 localhost.23424 > localhost.domain: 8438+[|domain]
02:38:22.014952 localhost.domain > localhost.34780: 56477 1/0/0 A mia09s01-in-f13.1e100.net(53)
02:38:22.609442 localhost.domain > localhost.23424: 8438[|domain]
02:38:22.610315 localhost.41377 > localhost.domain: 59282+ PTR? 77.2.217.172.in-addr.arpa.(43)
02:38:23.117080 localhost.domain > localhost.41377: 59282 2/0/0 PTR mia09s01-in-f13.1e100.net.,(112)
02:38:27.217860 localhost.13107 > localhost.domain: 60526+[|domain]
```

Ktrace
```
   870 iridium  CALL  sendto(52,0xe03b1de8000,0x5c0,0x400<MSG_NOSIGNAL>,0,0)
   870 iridium  GIO   fd 52 wrote 1472 bytes
       "\M-@\^E\0\0\^P\0\^A\0\0\0\0\0\0\0\0\0\^E\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\M-L&_\^U\M-H\M-s\M^S\M^T\M-X$(,\^]\M-QAM\^B\0\0\0\0\0\0\0\^B\0\0\0\0\0\0\0\M-If\M-c\^D\M-7\^Q\
        \^_^\^?5g\M-G\M-y\M-Y\M->JP\M-P\M-q\f4\^S\^?ka\^C\M-*9;\M-Y\^]D\M-If\M-c\^D\M-7\^Q\^_^\^?5g\M-G\M-y\M-Y\M->JDW|2&\M-]\M-Q+x\M^RWq|w:o\^A\0\0\0\0\0\0\0\^A\0\0\0\0\0\
        \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\M-If\M-c\^D\M-7\^Q\^_^\^?5g\M-G\M-y\M-Y\M->J\M^\[\M-U\M-@)\M-&G6\M^A\M-G\M-0\M^N\^?\M-/G\M-v\M-If\M-c\^D\M-7\^Q\^_^\^?5g\M-G\
        \M-y\M-Y\M->J\M-;C\M-GP\M-as\M-:\M-&\M^C~uf\^T]\M-Eu\^A\0\0\0\0\0\0\0\^A\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^F\^SN\M-V\M^\\M-U\M-Z\M-YR-|\b\M-l\M^^\M-N\
        \M-=Z\M-(\^W\M^@t%v\^ZD=\M-8H\^T\M-K5^\^B\0\0\0H\0\0\0\^A\0\0\0\^P\0\0\0\^A\0\0\0\0\0\0\0\^A\0\0\0\^P\0\0\0\^A\0\0\0\0\0\0\0\M-%i\M-@\M^H\^A\M-q4\^E\^A\0\0\0\0\0\0\
        \0\M-|\M-*\M^V\M-?\M^G\M^A\M-v\M-%\0\0\0\0\0\0\0\0\^X\0\0\0\0\0\0\0\0\0\0\0P\M-x\M-|Y\0\0\0\0\M-QL\M-~\M^?0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^X\0\0\0\0\
        \0\0\0\^A\0\0\0\0\0\0\0\b\^D\0\0\0\0\0\0\M-@\0\0\0\0\0\0\0\M-8\0\0\0\0\0\0\0\M-@\0\0\0\0\0\0\0(\^A\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^C\0\0\0\0\0\0\0\0\0\0\0@\^A\
        \0\0\0\0\0\0P\^A\0\0\0\0\0\0\^X\^B\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^A\0\0\0\0\0\0\0\^A\0\0\0\^B\0\0\0\0\0\0\0\0\0\0\0\b\^B\0\0\0\0\0\0\b\^B\0\0\0\0\0\0\M-~\M^?\M^?\M^?\
        \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\M-0\^B\0\0\0\0\0\0\M-@\^B\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\M-@\^B\0\0\0\0\0\0\f\0\0\0\^D\0\0\0POST\0\0\0\0\^P\0\
        \0\0\0\0\0\0\b\0\0\0\0\0\0\0]\0\0\0U\0\0\0https://accounts.google.com/ListAccounts?gpsia=1&source=ChromiumBrowser&json=standard\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\
        \0$\0\0\0\^\\0\0\0https://accounts.google.com/\0\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\^X\0\0\0\^B\0\0\0\^P\0\0\0\
        \0\0\0\0P\0\0\0\0\0\0\0\^X\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\^X\0\0\0\0\0\0\0\^N\0\0\0\^F\0\0\0Origin\0\0\^^\0\0\0\^V\0\0\0https://www.google.com\0\0\^X\0\0\0\0\0\0\0\
        \^P\0\0\0\0\0\0\0 \0\0\0\0\0\0\0\^T\0\0\0\f\0\0\0Content-Type\0\0\0\0)\0\0\0!\0\0\0application/x-www-form-urlencoded\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\
        \b\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0 \0\0\0\0\0\0\0\^X\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^P\0\0\0\^A\0\0\0\b\0\0\0\0\0\0\0X\0\0\0\0\0\0\0\^E\0\0\0\0\0\0\0H\
        \0\0\0\0\0\0\0P\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0X\0\0\0\0\0\0\0\M^?\M^?\M^?\M^?\0\0\0\0\M^?\M^?\M^?\M^?\0\0\0\0\0\0\0\0\0\0\0\0\^A\0\0\0\0\0\0\08\0\0\0\0\0\0\0    \0\0\
        \0\^A\0\0\0 \0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\b\0\0\
        \0\0\0\0\0\^P\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\b\0\0\0\0\0\0\0\^X\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\^X\0\0\0\0\0\0\0\^P\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0(\0\0\
        \0\0\0\0\0 \0\0\0\0\0\0\0(\0\0\0\0\0\0\0\M-;\^A\0\0\0\0\0\0\0\0\0\0\0\0\0\0\r\0\0\0\^E\0\0\0https\0\0\0\^[\0\0\0\^S\0\0\0accounts.google.com\0\0\0\0\0\^P\0\0\0\0\0\
        \0\0\M-1\M-0\^^\^B\0\0\0\0"
   870 iridium  RET   sendto 1472/0x5c0
   870 iridium  CALL  sendto(52,0xe037e8a9100,0x70,0x400<MSG_NOSIGNAL>,0,0)
```

This appears to be some sort of check as oppose to a persistent connection. Haven't taken the time to find the call in source and see why though. Seems to occur after loading the GTK+ Input Method Modules file. A few sysctl checks and many, many, gettime calls later...

**Comment by medozas** [comment]
Thanks for your report. I'm investigating this (which though might take some time) and get back to this ASAP

**Comment by jengelh** [comment]
Fixed in version 2020.11.85.
